### DISCUSSION TITLE
Can I use interpolation to generate the ownership field here?

### DISCUSSION THREAD

**Discussion by 4x4notfound** [cue-lang/cue#1167] [Q&A]
```
#capabilitySchema {
	owner: #user.id | #keyring.id | *"other"
	name: "user" | "group" | *"other" // TODO: user interpolation to generate?
	rights: #rights
	dek?: [...#dekSchema]
}
```

The goal here is the name property can be only a certain value based on the kind of owner.

**Comment by myitcv** [comment]
Thanks for raising as a discussion @maurerbot. I'm probably being a bit slow here, but can explain in a bit more detail what you're trying to achieve here?
